proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/workflow/WorkflowMaterialView.java

```diff
@@ -119,6 +120,10 @@ protected Canvas getExpansionComponent(final ListGridRecord record) {
         materialGrid.setCanSort(false);
         materialGrid.setCanGroupBy(false);
         materialGrid.setWrapCells(true);
+
+        CanvasSizePersistence persistence = new CanvasSizePersistence("WorkflowMaterialView.materialList", materialGrid);
```

Comment:
WorkflowMaterialView je komponenta použitá jak u panelu záměrů, tak úkolů. Je potřeba rozlišit prefix, jinak se budou rozměry vzájemně přepisovat.